source/includes/_payouts.md

```diff
@@ -196,6 +196,177 @@ Please note:
 If a request is being made using an IP address that is not in the whitelisted IP addresses,
 the server will respond with a [401 Unauthorized HTTP status code](https://tools.ietf.org/html/rfc7235#page-6).
 
+### Strong customer authentication
+
+There are some actions that require additional authentication.
+
+In those cases a 403 is returned together with a one-time token (OTT) value which needs 
+to be signed and the resulting signature included in the retry of the original request.
+
+We use a digital signature scheme based on public-key cryptography. It involves creating a
+signature using a private key on the client side and verifying the signature authenticity on
+the server side using the corresponding public key the client has uploaded.
+
+To call the endpoints requiring additional authentication:
+
+* Create a key pair
+* Upload the public key to TransferWise
+* Set up response handling to retry with the signed OTT
+
+**Creating the key pair**
+
+> Keys can be generated with the [OpenSSL](https://www.openssl.org/) toolkit:
+
+```bash
+$ openssl genrsa -out private.pem 2048
+$ openssl rsa -pubout -in private.pem -out public.pem
+```
+
+The following requirements apply:
+
+* The cryptographic algorithm is RSA.
+* Key length is at least 2048 bits.
+* The public key is stored in PEM file format using a `.pem` file extension.
+
+**Managing uploaded public keys**
+
+The public keys management page can be accessed from the Public keys section of the Settings page in your TransferWise
+account.
+
+While adding a new public key you'll be prompted for 2FA.
+
+Note that to prevent abuse the number of public keys you can have stored is limited to 5.
+
+**Signing the data**
+
+The digital signature algorithm we use is *SHA256 with RSA* (SHA256 hash of the data is signed with RSA). There are 
```

Comment:
I rephrased it this way:
```
We will only accept the signatures created with *SHA256 with RSA* (SHA256 hash of the data is signed with RSA) algorithm.
```
since we accept the signature itself, not the algorithm. WDYT?